daniel-robert/server.js

```diff
@@ -34,15 +36,15 @@ app.use(express.static('./public'));
 // REVIEW: Routes for requesting HTML resources
 app.get('/new', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Put your response here... 1) number 5.  2) Article.prototype.insertRecord() ) 3) R - it is reading data from the database.
```

Comment:
InsertRecord()? Think about when is this route being called?

Reply:
After looking back at the code, nothing in article.js interacts with this method directly.